tests/TestRunner.elm

```diff
@@ -3,11 +3,12 @@ port module TestRunner exposing (..)
 import Tests
 import Test.Runner.Node exposing (run, TestProgram)
 import Json.Encode exposing (Value)
+import Sample
```

Comment:
Ah apologies thats an leaked  experiment from my testing PR  that should not necessarily be linked to this.